### DISCUSSION TITLE
pycharm and clion exits immediately after full-screen mode

### DISCUSSION THREAD

**Discussion by cnut1648** [i3/i3#5715] [Q&A] [answered]
Hi, when I full-screen pycharm or clion using `fullscren toggle`, the full-screen mode will immediately exit. Interestingly, intellij does not have this issue. I wonder if anyone has pointer on why this is happening.

My i3 version is 4.22 (2023-01-02), and pycharm professional version is 2023.2.1.

To reproduce, use the simplest i3 config 
```
bindsym $mod+f fullscreen toggle
```
then open pycharm, use `$mod+f` then the window will immediately exit full-screen mode.

**Comment by orestisfl** [comment, marked as answer]
See https://github.com/i3/i3/issues/5695#issuecomment-1743707943

**Reply by cnut1648** [reply]
Hi @orestisfl , thanks for the info!
